unic/ucd/name/tests/basic_tests.rs

```diff
@@ -13,12 +13,37 @@ extern crate unic_ucd_name;
 use unic_ucd_name::Name;
 
 #[test]
-fn basic_tests() {
+fn test_name_str() {
     assert_eq!(
-        Name::of('A').expect("No name for A").to_string(),
+        Name::of('A').unwrap().to_string(),
         "LATIN CAPITAL LETTER A"
     );
+    assert_eq!(
```

Comment:
Uh, another nit: It's good practice to group the asserts together based on their rule in the spec, and have a comment for each group with the rule name. Like this:
```
// == NR1: Hangul Jamos ==

assert_eq!(
    ...
)
assert_eq!(
    ...
)

// == NR2: Ideographics ==

assert_eq!(
    ...
)
assert_eq!(
    ...
)

// == NR3: ... ==

assert_eq!(
    ...
)
assert_eq!(
    ...
)
```

Reply:
Fixed.